acl/acl.go

```diff
@@ -477,6 +481,22 @@ func (a *ACL) AllowQuotaWrite() bool {
 	}
 }
 
+// AllowPluginRead checks if read operations are allowed for all plugins
+func (a *ACL) AllowPluginRead() bool {
+	// ACL is nil only if ACLs are disabled
```

Comment:
This doesn't seem as good to me as having an explicit allow all ACL, but the pattern is pretty well set. Other methods exactly similar to this one like `AllowPluginQuota` don't do this nil check, but leave it to the calling code. I can't think of a concrete reason to do that.